schema/field.go

```diff
@@ -922,7 +923,7 @@ func (field *Field) setupValuerAndSetter() {
 					if !reflectV.IsValid() {
 						field.ReflectValueOf(ctx, value).Set(reflect.New(field.FieldType).Elem())
 					} else if reflectV.Kind() == reflect.Ptr && reflectV.IsNil() {
-						return
```

Comment:
I will raise a PR on the upstream for this. Once this is resolved I think we can go back to depending on the upstream version.